src/components/views/elements/AppTile.js

```diff
@@ -26,6 +26,7 @@ import { _t } from '../../../languageHandler';
 import sdk from '../../../index';
 
 const ALLOWED_APP_URL_SCHEMES = ['https:', 'http:'];
+const betaHelpMsg = 'This feature is currently experimental and is intended for beta testing only';
```

Comment:
This needs to be i18nised (and can't be done here because the language handler isn't ready at this point, but will be ready when rendering)